src/main/resources/db/changelog/changes/v1.0.3/sql/create-list-versions-table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS list_versions
+(
+    id                      UUID            NOT NULL PRIMARY KEY,
+    name                    VARCHAR(255)    NOT NULL,
+    description             VARCHAR(1024),
+    entity_type_id          UUID            NOT NULL,
+    fql_query               VARCHAR(1024),
+    user_friendly_query     VARCHAR(1024),
+    fields                  VARCHAR[],
+    updated_by              UUID            NOT NULL,
+    updated_by_username     VARCHAR(64)     NOT NULL,
```

Comment:
Where'd this 64 come from? It needs to be at least as big as what's supported by mod-users, so I just want to make sure.